backend/homework/homework-3/src/main/java/kdu/quesone/User.java

```diff
@@ -0,0 +1,52 @@
+package kdu.quesone;
+
+public class User {
+    private long id;
+    // add rest of the variables
```

Comment:
Clear dangling comments before final push. Unless required.
Just to maintain clean code